pkg/trace/pb/stats.proto

```diff
@@ -15,38 +15,45 @@ message StatsPayload {
 	bool clientComputed = 5;
 }
 
-// TODO(gbbr): doc
+// ClientStatsPayload is the first layer of span stats aggregation. It is also
+// the payload sent by tracers to the agent when stats in tracer are enabled.
 message ClientStatsPayload {
-	string hostname = 1;
-	string env = 2;
-	string version = 3;
+        // hostname is the tracer hostname. It's extracted from spans with "_dd.hostname" meta
+        // or set by tracer stats payload when hostname reporting is enabled.
+	string hostname = 1; // env tag set on spans or in the tracers, used for aggregation
+	string env = 2; // env tag set on spans or in the tracers, used for aggregation
+	string version = 3; // version tag set on spans or in the tracers, used for aggregation
 	repeated ClientStatsBucket stats = 4 [(gogoproto.nullable) = false];
-	string lang = 5;
-	string tracerVersion = 6;
-	string runtimeID = 7;
-	uint64 sequence = 8;
+	string lang = 5; // informative field not used for aggregation
+	string tracerVersion = 6; // informative field not used for aggregation
+	string runtimeID = 7; // used on stats payloads sent by the tracer to identify uniqu
+	uint64 sequence = 8; // informative field not used for aggregation
+        // agentAggregation is set by the agent on tracer payloads modified by the agent aggregation layer
+        // characterizes counts only and distributions only payloads
+	string agentAggregation = 9;
 }
 
-// TODO(gbbr): doc
+// ClientStatsBucket is a time bucket containing aggregated stats.
 message ClientStatsBucket {
-	uint64 start = 1;
-	uint64 duration = 2;
+	uint64 start = 1; // bucket start in nanoseconds
+	uint64 duration = 2; // bucket duration in nanoseconds
 	repeated ClientGroupedStats stats = 3 [(gogoproto.nullable) = false];
+	int64 agentTimeShift = 4; // bucket start timeshift applied by the agent client stats aggregator
```

Comment:
nit: could be nice to add something like "if the payloads is received outside of the agent aggregation window" to give more context on when this is done.